src/Propel/Generator/Platform/MysqlPlatform.php

```diff
@@ -717,10 +718,96 @@ public function getAddColumnDDL(Column $column)
         );
     }
 
+    public function getModifyTableDDL(TableDiff $tableDiff)
+    {
+        $alterTableStatements = '';
+
+        $toTable = $tableDiff->getToTable();
```

Comment:
this var is only used in one certain case below, therefore I would move it more to this location.